src/app/shared/components/users-billing-info/users-billing-info.component.html

```diff
@@ -125,10 +126,17 @@ <h3 class="form-step-card-title text-dark mb-3">
                         <strong>{{upgradeAmount|currency:currency}}</strong>
                       </div>
                     </div>
-
+                  </div>
                 </div>
               </div>
-              <div class="form-step-card" *ngIf="paymentMethod===paymentMethodType.BITCOIN && planType!==planTypeEnum.FREE">
+              <div class="monero"*ngIf="Monero">
+                <h1 class="moneroInfo">Payment Information</h1>
```

Comment:
for the class attribute, it should not be `camelCase`.
write like this, `monero-info`